frontend/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Use the official Python image from the Docker Hub
+FROM python:3.9-slim
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the requirements file into the container
+COPY requirements.txt .
```

Comment:
Why is this not using the uv environment?

Reply:
I just knew the syntax using the requirements file off the top of my head - plus since we're building each service within a docker container, there's no need for the overhead of building a whole python package!